src/Arcus.Messaging.Tests.Workers.ServiceBus/Program.cs

```diff
@@ -24,22 +30,40 @@ public static void Main(string[] args)
                 {
                     configuration.AddCommandLine(args);
                     configuration.AddEnvironmentVariables();
+                    configuration.AddInMemoryCollection(new []{ new KeyValuePair<string, string>("ARCUS_HEALTH_PORT", "5000") });
                 })
+                .ConfigureLogging(loggingBuilder => loggingBuilder.AddConsole(options => options.IncludeScopes = true))
                 .ConfigureServices((hostContext, services) =>
                 {
+                    services.AddSingleton<ISecretProvider>(serviceProvider =>
+                    {
+                        var authentication = new ServicePrincipalAuthentication(
+                            "d1c9a6b8-e26e-4bd7-85a7-98140ce73cb1",
+                            "B7y/GrQV6T.74UZpNBxGose=fIBEqa=W");
```

Comment:
Please move this one back to the variable group; I used a temp one and didn't have this value anymore (and too lazy for LastPass 😅)